### ISSUE TITLE
Backlight widget not changing brightness

### ISSUE THREAD

**Issue by nullbyto** [qtile/qtile#3728]
### The issue:

Brightness percentage is shown but not changing when scrolling.
```
2022-07-28 01:36:56,587 ERROR libqtile loop.py:_handle_exception():L63 Command '['xbacklight', '-set', '55.0']' returned non-zero exit status 1.
```

When i try setting brightness from terminal using `xbacklight -set 1`:
```
No outputs have backlight property
```

Note i change brightness using `brightnessctl`.

`Qtile version: 0.21.0`

### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by elParaguayo** [comment]
That's not a qtile issue. You need to find out why xbacklight isn't working. Try searching for that error message.

**Comment by nullbyto** [comment]
Yes you are right, i thought maybe someone knows about that problem.
Anyways i tried a lot of potential solutions which doesn't really work in my case with `xbacklight` it has something to do with my intel interface.

Light or brightnessctl works for me only.

Maybe i could change the `change_command` key for `Backlight` but can't manage to get it working since `brightnessctl` has different argument syntax than `xbacklight`.

TL;DR: A simple solution for anyone having the same problem set this for the `Backlight` widget:
```
mouse_callbacks={
    "Button4": lambda: qtile.cmd_spawn("brightnessctl set +5%"),
    "Button5": lambda: qtile.cmd_spawn("brightnessctl set 5%-"),
}
```
